src/website/src/app/documentation/demos/a11y/accessibility.demo.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component } from '@angular/core';
+import { ClarityDocComponent } from '../clarity-doc';
+
+const EXAMPLE1 = `
+import { Component, AfterViewInit, AriaLiveService } from '@angular/core';
+
+@Component({
+  selector: 'my-component',
+  providers: [AriaLiveService],
```

Comment:
If it is provided in the root, then you don't need to define a provider, update throughout.

Reply:
In a way, it could be, but we need to change how the service work internally.  A new ID for the aria-live container will be created only when a new instance is made. If we put it as `root` injectable we gonna have only one container to be used across the app. This could be ok, but I never tested it. It may have some strange behavior when there is a mix of politeness types, for example, one message is `polite` other `assertive` and there close in time to each other. Not sure what will be the end result when it comes to reuse the container.

In a way, it will be a little more DOM efficient to use only one container ... but this may not work well when it comes to a lot of messages or mixed types. 

This is the internal reason to provide it for every component that will require it. 

Adding only `@Injectable` on top of the service won't do the job.


If we want to change this now is the time?